Exercise_1_Using_Loop.java

```diff
@@ -0,0 +1,38 @@
+package com.precourse2;
+
+public class BinarySearchAlgoUsingLoop {
+
+	// Returns index of x if it is present in arr[l.. r], else return -1
+	int binarySearch(int arr[], int l, int r, int x) {
+		// Write your code here
+		if (arr[r / 2] == x) {
+			return arr[r / 2];
+		} else if (arr[r / 2] > x) {
+			for (int i = 0; i < r / 2; i++) {
+				if (arr[i] == x) {
+					return i;
+				}
+			}
+		} else {
+			for (int j = r / 2; j <= r; j++) {
+				if (arr[j] == x) {
+					return j;
+				}
+			}
+		}
+		return -1;
+	}
```

Comment:
This is not binary search because we are iterating through half the list every time.

Reply:
ohh we have iterate through 2 elements also sometime or more elements, not complusorly half loop everytime.

Reply:
We should reduce the input list to half after every iteration until we find the required element

Reply:
yes got it
I have posted recursive method also for PR